installbuilder/datafiles/Base_DSC.data

```diff
@@ -194,6 +195,16 @@ chmod 700 $CONFIG_SYSCONFDIR/${{SHORT_NAME}}
 mkdir -p /var/opt/microsoft/omsconfig
 chown omsagent /var/opt/microsoft/omsconfig
 chgrp omsagent /var/opt/microsoft/omsconfig
+chmod 755 /var/opt/microsoft/omsconfig
+
+# Set up log file
+if [! -f /var/opt/microsoft/omsconfig/omsconfig.log ]
```

Comment:
@kwirkykat  Should we also do this for the detailed log?

Reply:
Done